File: python_project/oops/Banking/PersonalCustomer.py
from Customer import Customer 

class PersonalCustomer(Customer): 
    def __init__(self, name: str):
        super().__init__(name)  
        self.__address = None
        self.__home_phone = None
        self.__work_phone = None

    def get_address(self) -> str:
        return self.__address

    def set_address(self, address: str) -> None:
        self.__address = address

    def get_home_phone(self) -> str:
        return self.__home_phone    

    def set_home_phone(self, home_phone: str) -> None:
        self.__home_phone = home_phone

    def get_work_phone(self) -> str:
        return self.__work_phone

    def set_work_phone(self, work_phone: str) -> None:
        self.__work_phone = work_phone

File: java_project/oops/CashForMetals/PersonalCustomer.java
package oops.CashForMetals;


public class PersonalCustomer extends Customer {
    private String address;
    private String homePhone;
    private String workPhone;

    public PersonalCustomer(String name) {
        super(name); // Takes 1 String [cite: 24]
    }

    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }

    public String getHomePhone() { return homePhone; }
    public void setHomePhone(String homePhone) { this.homePhone = homePhone; }

    public String getWorkPhone() { return workPhone; }
    public void setWorkPhone(String workPhone) { this.workPhone = workPhone; }

    @Override
    public String toString() {
        return super.toString() + " (Personal) | Address: " + address;
    }
}
